### DISCUSSION TITLE
AlDente Free - does not stop at designated % and does not allow charging

### DISCUSSION THREAD

**Discussion by gpytel** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1222] [Q&A]
Hello! I'm considering AlDente Pro, but before I do I'm hoping someone can help me figure out some settings.

When I have it on and the cable is connected - the % goes up, but the comment from Apple says "battery not charging" - is this ok? Is it charging or not, since the power goes up?

Also, I don't know how to set up the lowest charge (the 20%)? That means the battery goes down anyway, even when the AlDente is running.

The third problem is - I have set up 78% as the top charge, but the batter goes over anyway, even when AlDente is running.

Am I doing something wrong? Or are does only in the Pro version?

In advance, thank you for all the help - I'm a noob, so please be gentle ;-)

**Comment by MatthiasKerbl** [comment]
Hi @gpytel,

Thank you for considering AlDente Pro.
Please answer the following questions so that I can help you better.

- Which MacBook model do you have?
- Which macOS version do you have installed?
- Which AlDente version do you have installed?

Also, please check out our [blog](https://apphousekitchen.com/blog/) where we have detailed descriptions of the features of AlDente.

Best,
Matthias

**Reply by gpytel** [reply]
Hi Matthias! Thank you for reaching out :-)

I have the Macbook Air M2, with OS Sonoma, and AlDente 1.26.1.

I did read the blog and FAQ, must a lot of them I don't really understand, to be honest... I really an a noob (and we know Macbook alllows noobs to feel important ;-) )

**Reply by MatthiasKerbl** [reply]
Hi @gpytel,

No worries, I will try to answer your questions but also keep in mind that we offer a 14-day money-back guarantee if you want to try it out.
In general, due to technical limitations, your MacBook will always charge to 100% when you turn it off. If you activate the feature "Stop charging while sleeping", the change limit will persist during sleep. If you have "Disable Sleep until charge limit" activated, sleep will be disabled until the change limit is reached, afterward, your MacBook will go to sleep and the charge limit will persist. I am not sure what exactly you mean by "Also, I don't know how to set up the lowest charge (the 20%)? That means the battery goes down anyway, even when the AlDente is running.". However, it is a common misconception that the Sailing Mode does actively charge and discharge your MacBook between the sailing range. This would actually be much worse for the battery. Instead, AlDente lets your battery slowly "fall back" to the lower limit, which you chose in the settings.

Hope that answers some of your questions

Best,
Matthias

**Reply by gpytel** [reply]
Thank you, Matthias! Some of what you said does make sense! I will now review the settings for sleeping and observe what happens.

The Sailing Mode is I think what I still struggle with, and let me try to explain what I meant. Is Sailing Mode the name of the function that let's me set up the lowest % of batter I'd like to use and maybe give me a warning? 
Currently, I only can set up the top level of % - so, ex. 80% - and when the laptop charges, it stops charging at 80% and keeps the charge (I think that's how it should be, correct?). 
However, I don't know where to set up what the low charge is, so my laptop - or AlDente? - gives me a warning. 
If the recommended charge range is "not lower than 20% and not higher ythan 80%" - how do I manage the "not lower than 20%" bit? I can't figure this out, therefore when I'm on battery, it does drain lower than 20% . Should I manage that as well, somehow? Where exactly, or is it reserved for AlDente Pro?

I don't understand this: "However, it is a common misconception that the Sailing Mode does actively charge and discharge your MacBook between the sailing range." :-)

Thank you for your help and patience <3

**Reply by MatthiasKerbl** [reply]
Hi @gpytel,

In general, you can only set one limit in AlDente, the Charge Limit, which is the percentage you want your battery to stay at. With Sailing Mode, you can also set a Sailing Range. When the battery "falls back" to the lower end of this range, the battery will charge to the charge limit again. You can read more about this feature [here](https://apphousekitchen.com/feature-explanation-sailing-mode/).
AlDente does currently not offer to set a lower limit to notify you when your battery is at for example 20%.

Best,
Matthias

**Comment by gpytel** [comment]
Understood! Thank you! I think this concludes this thread :-)
